lib/core/utilities.js

```diff
@@ -110,3 +110,8 @@ export function getProp (holder, propName) {
 
   return result
 }
+
+// Ie "Bearer " + token
+export function addTokenPrefix (token, tokenType) {
+  return tokenType ? tokenType + ' ' + token : token
```

Comment:
probably we can add a startsWith check here to prevent adding double prefix?

Reply:
I think it's not necessary because if `tokenType` is undefined it prevent from adding prefix.

Reply:
undefined token type is handled but what if token already has a prefix? Previously we had a similar bug so now that we refactor to a function, we can also ensure that doesn't happens :)

Reply:
Ok! Got ya!

Reply:
Improved in commit https://github.com/nuxt-community/auth-module/pull/548/commits/0dc1af3a5a2a3695a5e7b5c52c8a05514cc5382f